src/Dataset.ts

```diff
@@ -407,28 +447,14 @@ export default class Dataset {
     const quadsString = new n3.Writer().quadsToString(quads); // This always returns nquads or ntriples. See https://github.com/rdfjs/N3.js/issues/253
     const tmpFile = path.resolve(tmpdir(), `triplydb-${md5(quadsString)}.nq`);
     await fs.writeFile(tmpFile, quadsString, "utf-8");
-    return this.importFromFiles(opts || {}, tmpFile);
+    return this.importFromFiles([tmpFile], opts || {});
   }
-  public async importFromUrls(...urls: string[]): Promise<Dataset>;
```

Comment:
@RossCurry I like the improvements to `importFrom<x>`. But, the `importFromStore` interface is inconsistent now